### ISSUE TITLE
Firejailed Gajim doesn't start on Debian 9

### ISSUE THREAD

**Issue by roadkillazpartybird** [netblue30/firejail#1059]
Hi! 

Debian 9 GNOME
firejail version 0.9.44.4
gajim version 0.16.6-1

It runs fine without firejail, After running 'firejail gajim' it fails to open with error message:

> 
> Disk Write Error:
> [Errno 13] Permission denied: u'/home/bigeagle/.cache/gajim/gajim.pid'
> 

I tried adding these 2 lines to the profile with no success:
whitelist ${HOME}/.cache/gajim/gajim.pid
whitelist /home/bigeagle/.cache/gajim/gajim.pid

I also tried altering the permissions of that file to 666 instead of 644, no luck.

Any ideas on what might be wrong?

**Comment by netblue30** [comment]
I'll take a look, thanks for the bug.

**Comment by trustingfrust** [comment]
gajim.profile is missing
```
noblacklist ${HOME}/.cache/gajim
noblacklist ${HOME}/.local/share/gajim
noblacklist ${HOME}/.config/gajim
```
for these directories that are blacklisted in disable-programs.inc.

**Comment by netblue30** [comment]
The problem is on 0.9.44-bugfix branch only, on mainline is fine. I put a fix in 0.9.44-bugfix branch, thank you for the bug.
